nx/source/services/nifm.c

```diff
@@ -119,6 +124,40 @@ Result nifmIsWirelessCommunicationEnabled(bool* out) {
     return rc;
 }
 
+Result nifmSetWirelessCommunicationEnabled(bool enable) {
+    IpcCommand c;
```

Comment:
Add a check to make sure that this function returns `MAKERESULT(Module_Libnx, LibnxError_NotInitialized)` if the privilege level is incorrect (otherwise we risk IPC session being forcefully closed)